lib/iris/tests/integration/test_pp.py

```diff
@@ -47,6 +47,8 @@ def _test_coord(self, cube, point, bounds=None, **kwargs):
         if bounds is not None:
             self.assertArrayEqual(coords[0].bounds, [bounds])
 
+    # hits a segfault, very odd
```

Comment:
These should be fixed by : #2395
Short version: since we changed our test for lazy data, a simple Mock tests as lazy instead of real.
Then ```np.ma.asarray()``` can hit an endless loop (whereas ```np.asarray``` was ok).